src/ansible_navigator/configuration_subsystem/navigator_post_processor.py

```diff
@@ -647,7 +647,7 @@ def _normalize_path(
         exit_messages: list[ExitMessage] = []
 
         if entry.value.current is not C.NOT_SET:
-            normalized = Path(entry.value.current).resolve(strict=strict_resolve).as_posix()
```

Comment:
I see it problematic to resolve to path that does not exist. In most cases this is a configuration mistake and once that is likely to cause other problems.

Still, I can understand that in some cases, this might happen without being a show-stopper (maybe linux paths that do not exist on macos?). For this reason I suggest this, try using strict, catch the exception as a warning message, log it and fallback to non strict one. This should make it work without having to add other configuration options.